source/components/Layout/TxLayout.tsx

```diff
@@ -22,50 +22,45 @@ interface ITxLayout {
 }
 
 export const TxLayout: FC<ITxLayout> = ({ confirmRoute, txType, title }) => {
-  const controller = useController();
+  const accountCtlr = useController().wallet.account;
```

Comment:
```suggestion
  const accountController = useController().wallet.account;
```

we don't need to abbreviate variable names, it is more difficult to read